tests.cpp

```diff
@@ -29,16 +30,74 @@ int mcp23017_test() {
 
   return 0;
 }
+int gpio_test(Serial serial) {
+  int returnCondition = 0;
+  // initialize the pins
+  Gpio inputTest(0, 1);  // pin number is currently a dummy number

```

Comment:
Let's define macros at the top of the file for the pin numbers so we can easily change them as needed